source/partials/_progress-bars.html.haml

```diff
@@ -0,0 +1,5 @@
+%h5.progress-header= language.name
```

Comment:
We need to restructure this markup so that each rendered progress bar section is contained inside its own parent element. Maybe use a section tag? That makes it easier to target each progress bar for styling later.

Use [BEM syntax](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) for this.

Example from my blog: https://github.com/joshukraine/ofreport.com/blob/4794eb1c160382acb3ebc6399446c9ceedec7184/source/index.html.haml#L10